draft-ietf-quic-transport.md

```diff
@@ -778,6 +783,33 @@ are in a different packet number space to other packets (see
 {{packet-numbers}}).
 
 
+### 0-RTT Packet Numbers {#retry-0rtt-pn}
+
+Packet numbers for 0-RTT protected packets use the same space as 1-RTT protected
+packets.
+
+After a client receives a Retry or Version Negotiation packet, 0-RTT packets are
+likely to have been lost or discarded by the server, especially if the server
+changes its connection ID with a Retry packet.  A client MAY attempt to resend
```

Comment:
The text after "especially if ..." seems to imply that 0-RTT packets might not have been lost in case the client receives a Version Negotiation packet.

I am not sure if that is true.

My understanding is that a Version Negotiation packet is sent by a server only if the client offers a version unknown to the server. In such case, all the 0-RTT packets would have been lost, because the format of the 0-RTT packets are version specific, and those would have been unprocessible by the server.

Reply:
Yeah, this is right.  In order of likelihood that packets are unusable:

1. VN with unsupported version
2. Retry w/ connection ID change
3. Retry w/o connection ID change or VN with preferred alternative version

I don't know why I missed that.  Reworded.